conrad/cli.py

```diff
@@ -185,18 +191,48 @@ def _show(ctx, *args, **kwargs):
         events_output = []
 
         for event in events:
-            events_output.append(
-                [
-                    event.id,
-                    event.name,
-                    event.url,
-                    event.city,
-                    event.state,
-                    event.country,
-                    event.start_date.strftime("%Y-%m-%d"),
-                    event.end_date.strftime("%Y-%m-%d"),
-                ]
-            )
+            if(reminders):
+                for reminder, __ in reminders:
+                    if(reminder.id==event.id):
+                        events_output.append(
+                        [
+                            Fore.GREEN + Style.BRIGHT + event.id + Style.RESET_ALL,
```

Comment:
Just highlighting the id with "yellow" should be good.

Reply:
updated